packages/components/pie-icon-button/src/defs.ts

```diff
@@ -1,7 +1,18 @@
 export interface IconButtonProps {
+    size: 'xsmall' | 'small' | 'medium' | 'large';
     variant: 'primary' | 'secondary' | 'outline' | 'ghost' | 'ghost-tertiary';
 }
 
+/**
+ * Icon Button size variants
+ */
+export const iconButtonSizes: IconButtonProps['size'][] = [
```

Comment:
I think this might be a bit more readable, if it works?
```suggestion
export const iconButtonSizes: Array<IconButtonProps['size']> = [
```

Reply:
I'd probably suggest if we change this, we do it across all defs as one change (as it was updated everywhere by the ESLint changes). I don't mind either way, but think it should be consistent across all components.

@jamieomaguire @raoufswe any thoughts? Happy to do this in a separate PR if we think the change is beneficial.

Reply:
Agreed the above looks easier to read (at least for me from a non-ts user perspective). Happy for this to be done in one go rather than this pr

Reply:
I like Xander's suggestion! A separate one-bang PR for all of them makes sense. @ashleynolan could you raise that after this please?

Reply:
Sure – I'll do that once this is merged 👍